### ISSUE TITLE
Custom Date

### ISSUE THREAD

**Issue by rzarai** [herdsothom/joplin-insert-date#3]
Can I insert custom date?  
I ask for this feature since I use date format that looks like this: 21y09m28d  
Or this: 21y-09m-28d

**Comment by Brandonmccoub2** [comment]
Yeah I would like this too, not sure if there is a way to do this.

**Comment by Domoel** [comment]
would be great if  we could adjust the way how its rendered (e.g. DD.MM.YYYY or DD/MM/YYYY or MM/DD/YYYY, etc.)
